### DISCUSSION TITLE
참고하면 좋은 아트

### DISCUSSION THREAD

**Discussion by fkdl0048** [GG-Studio-990001/Grayed-Game#44] [Ideas]
- [2.5D? 사이드 뷰](https://www.reddit.com/r/Unity2D/comments/173duiy/background_for_my_game_what_you_think/)
- [2D 쿼터?](https://www.reddit.com/r/Unity2D/comments/1769lf0/i_cant_decide_on_what_version_is_better_any/)

![image](https://github.com/GG-Studio-990001/GameOver/assets/84510455/56b103d2-7a89-4207-8368-df305831647e)

![image](https://github.com/GG-Studio-990001/GameOver/assets/84510455/7913c6f9-ac3b-4365-af57-e2f51dc6c0d5)

![Uploading image.png…]()

- [3D](https://www.pinterest.co.kr/pin/180777372534509901/)

돌아다니다 생각보다 이뻐서,,

**Comment by fkdl0048** [comment]
[마음에 드는 UI](https://www.pinterest.co.kr/pin/61572719897507888/)

**Comment by fkdl0048** [comment]
[참고하면 좋을 듯한 시스템?](https://www.reddit.com/r/godot/comments/17fs2k9/first_fan_game_with_godot_polish_really_makes_a/)

**Comment by fkdl0048** [comment]
![image](https://github.com/GG-Studio-990001/GameOver/assets/84510455/4798d445-a99c-473b-93ad-d37afc098cae)

**Reply by fkdl0048** [reply]
![image](https://github.com/GG-Studio-990001/GameOver/assets/84510455/1292a37e-08ae-40bc-823d-d1274da267ed)

**Reply by fkdl0048** [reply]
![image](https://github.com/GG-Studio-990001/GameOver/assets/84510455/f4ca5848-fa61-4ef1-a59e-ba5dc7614ac8)

**Comment by fkdl0048** [comment]
[깔끔한 애니메이션](https://www.reddit.com/r/godot/comments/17h3gjd/break_the_ice/)

**Comment by fkdl0048** [comment]
https://www.reddit.com/r/Unity2D/comments/181ajeg/made_a_free_dungeon_tileset_you_can_use_on_any/

@yanggang3 

회의 때 말씀하신 게 이게 맞나요?

**Reply by yanggang3** [reply]
@fkdl0048 엇 확인이 늦어 죄송합니다.!
혹시 바닥의 색이나 생김새를 말하는 걸까요? 아니면 지금 보여주는 분위기(던전,어두운?) 를 말씀하시는 걸까요?
저는 팩맘이 청소하는 주부를 모티브로 약간 실내 집같은 우드색이 많은 이런 느낌으로 말한거였어요!
![At Home Alone Final - How to Solve Stone Brick Puzzle](https://github.com/GG-Studio-990001/GameOver/assets/145998716/5febcb8c-d5b0-4112-b71f-e016ea130d42)
![Комната в доме ториель](https://github.com/GG-Studio-990001/GameOver/assets/145998716/061be006-ccf9-4423-b9a9-0286b6e85256)
![다운로드3](https://github.com/GG-Studio-990001/GameOver/assets/145998716/9b83b943-fe26-4a04-b721-66b4d7885982)

보여주신 방향으로 가는 것도 어울리는 것 같습니다,,

**Reply by fkdl0048** [reply]
@royalbluesm 악! 태그를 잘못 했네요!

민지님이 그림으로 보여주신 조명효과 말씀드린 겁니다!

논외로 레퍼런스 좋네요!

**Reply by royalbluesm** [reply]
;; 태그 알림 확인이 늦어졌네요 죄송합니다!! 
조명효과는 있으면 좋고 없으면 정지 이미지로만 나타내도 괜찮아요~ 제가 말한 건 저거 맞습니다!
